packages/matchbox/src/components/Snackbar/Snackbar.tsx

```diff
@@ -22,8 +20,23 @@ const StyledClose = styled(Box)`
   ${dismissStatus}
 `;
 
-const Snackbar = React.forwardRef(function Snackbar(props, userRef) {
-  const { children, status, maxWidth, onDismiss, 'data-id': dataId, ...rest } = props;
+type SnackbarProps = {
+  'data-id'?: string;
+  status?: 'default' | 'success' | 'danger' | 'warning' | 'error';
+  onDismiss?: (any) => void;
```

Comment:
This technically isn't `any` right? We pass through the click event. From googling, I think it should be `onDismiss?: React.MouseEventHandler<HTMLButtonElement>;`

Reply:
or `(event: React.MouseEvent ) => void;`